src/Ticket.php

```diff
@@ -2445,6 +2445,8 @@ public function updateDateMod($ID, $no_stat_computation = false, $users_id_lastu
                         '_disablenotif'              => true
                     ]
                 );
+            } else if ($no_stat_computation) {
+                return;
```

Comment:
I do not understand why you are blocking the update of the `date_mod`/`users_id_lastupdater` fields here.